core/src/main/java/de/danoeh/antennapod/core/preferences/UserPreferences.java

```diff
@@ -1057,4 +1059,15 @@ public static void setFeedFilter(String value) {
                 .apply();
     }
 
+    public static long getUsageCountingMillis() {
```

Comment:
Please add something like "date" to the name. It currently sounds like it returns the millis between today and the first install.